pandas/tests/indexing/test_scalar.py

```diff
@@ -185,6 +186,17 @@ def test_at_with_tz(self):
         result = df.at[0, 'date']
         assert result == expected
 
+    @pytest.mark.parametrize('tz', [
+        None, 'UTC', 'US/Eastern', 'Asia/Tokyo',
+        'dateutil/US/Pacific', 'CET', FixedOffset(60)])
+    def test_series_set_tz_timestamp(self, tz):
```

Comment:
Use the `tz_naive_fixture` (i.e. `def test_series_set_tz_timestamp(self, tz_naive_fixture)`) instead of specifying all these tzs

Reply:
Oh sorry, did not now how it worked. Thanks will do that right now!